calico/calico.go

```diff
@@ -52,6 +52,10 @@ func CreateOrUpdateGlobalNetworkSet(client client.Interface, name string, labels
 		metrics.IncCalicoClientRequest("create", err)
 		return err
 	}
+	if err != nil {
+		metrics.IncCalicoClientRequest("get", err)
+		return err
+	}
```

Comment:
I don't see a `metrics.IncCalicoClientRequest("get", err)` in the case where `err == nil` or `err == errors.ErrorResourceDoesNotExist`. I would expect the get request to be recorded, regardless of whether there was an error or not.

Reply:
hmm basically what I am trying to avoid here is recording a get request with an error while I expect that there might be one. I think you have a point though and I should be adding a `metrics.IncCalicoClientRequest("get", nil)` above to capture the call without any error